tests/framework/ROM/TimeSeries/pickled_arma.xml

```diff
@@ -0,0 +1,75 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries.ARMA_pickled</name>
+    <author>talbpaul</author>
+    <created>2018-01-12</created>
+    <classesTested>SupervisedLearning.ARMA</classesTested>
+    <description>
+      Assures that a pickled (serialized) ARMA samples identically to the original model
+      if reseedCopies is False.
```

Comment:
where does this keyword (reseedCopies) need to be mentioned? 
Isn't the default for this "True"?

Reply:
Default is "False", and it's in the manual already.